main.cpp

```diff
@@ -1,7 +1,135 @@
+/*
+
+Current Problem:
+
+How can I exclude the already filled elements so they cannot be overwritten, without using multiple (if else) statements...
+
```

Comment:
have a second array bool check[9] (same size) to store if a space is used already, check this anytime the user puts the input